src/styles/button.scss

```
@@ -0,0 +1,218 @@
+}
+
+.place-card-text{
+    position: absolute;
+    text-align: center;
+    width: 100%;
+    top: 100%;
+    left: 50%;
+    color: white;
+    opacity: 0;
+    font-family: $font-family;
+    transform: translate(-50%, -50%);
+    transition: all ease-in-out 0.35s;
+}
+
+.plan-card-text{
+    position: absolute;
+    text-align: center;
+    width: 100%;
+    top: 100%;
+    left: 50%;
+    color: white;
+    opacity: 0;
+    font-family: $font-family;
+    transition: all ease-in-out 0.35s;
+}
+
+.detail-text{
+    position: absolute;
+    text-align: right;
+    top: 88%;
+    width: 90%;
+    left: 50%;
+    font-size: 10px;
+    padding-left: 50px;
+    margin: 0 auto;
+    color: white;
+    opacity: 0;
+    font-family: $font-family;
+    transform: translate(-50%, -50%);
+}
+
+.plan-owner-text{
+    position: absolute;
+    text-align: left;
+    top: 95%;
+    width: 90%;
+    font-size: 10px;
+    margin: 0 auto;
+    color: white;
+    opacity: 0;
+    font-family: $font-family;
+}
+
+/* recommened place card properties */
+
+.rec-place-card{
+    position: relative;
+    width: 300px;
+    height: 300px;
+    float: left;
+    opacity: 0.9;
+    filter: drop-shadow(0px 4px 4px rgba(0, 0, 0, 0.25));
+    border-radius: 20px;
+    border-style: none;
+    background-image: url("https://source.unsplash.com/300x300/?travel");
+    background-position: center;
+    background-size: cover;
+    overflow: hidden;
+    cursor: pointer;
+}
+
+.rec-place-card:hover .place-card-text{
+    top: 50%;
+    left: 50%;
+    opacity: 1;
+    transform: translate(-50%, -50%);
+}
+
+.rec-place-card .card-overlay{
+    position: absolute;
+    width: 100%;
+    height: 100%;
+    top: 0;
+    bottom: 0;
+    left: 0;
+    right: 0;
+    background: rgba(0, 0, 0, 0.8);
+    opacity: 0;
+    transition: all ease-in 0.3s;
+}
+
+.rec-place-card:hover .card-overlay {
+    opacity: 1;
+}
+
+
+/* recommened plan Card properties */
+
+.rec-plan-card{
+    position: absolute;
+    width: 263px;
+    height: 600px;
+    float: center;
+    opacity: 0.8;
+    background: url("https://source.unsplash.com/263x600/?travel"),
+        linear-gradient(180deg, rgba(196, 196, 196, 0) 0%, 
+        rgba(0, 0, 0, 0.8) 100%);
+    border-radius: 20px;
+    border-style: none;
+    background-position: center;
+    background-size: cover;
+    overflow: hidden;
+    cursor: pointer;
+    transition: all ease-in 0.3s;
+}
+
+
+.card-overlay{
+    position: absolute;
+    width: 100%;
+    height: 100%;
+    background: linear-gradient(180deg, rgba(196, 196, 196, 0) 0%, rgba(0, 0, 0, 0.5) 100%);
+    opacity: 0;
+    transition: all ease-in 0.3s;
+}
+
+.rec-plan-card .plan-card-text{
+    top: 80%;
+    width: 90%;
+    padding-left: 50px;
+    padding-right: 20px;
+    text-align: right;
+    font-size: 10px;
+    margin: 0 auto;
+    z-index:10;
+    opacity: 1;
+    transform: translate(-50%, -50%);
+}
+
+.rec-plan-card:hover {
+    width: 284px;
+    height: 650px;
```

use `transformation` instead of increasing the width and height it will increase the size or transform everything under this class including itself
```suggestion
.rec-plan-card:hover {
    transform: scaleY(1.08);
```